release/test-dependencies/expected/test_release_gold_build.properties

```diff
@@ -0,0 +1,32 @@
+
+	plugins.war.zip.url[2024.q3.12]=http://release-1/1/userContent/liferay-release-tool/7413/plugins.war.latest.zip
```

Comment:
@lucasmiranda0, why did you need to create this new file? Was it really intended?